From 047e8305093ee38d89488d7470fa7175087877f2 Mon Sep 17 00:00:00 2001
From: cloud-comms-engine <engine@diegonmarcos.com>
Date: Tue, 23 Jun 2026 14:13:46 +0300
Subject: [PATCH] comms: rename app to Cloud Mail (Fairmail)

Single-line app_name change (strings.xml). All other 'FairEmail' literals
are log tags / internal magic headers (backup compat) and are intentionally
left untouched.
---
 app/src/main/res/values/strings.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/src/main/res/values/strings.xml b/app/src/main/res/values/strings.xml
index 06903e8d6e..5cff760a2c 100644
--- a/app/src/main/res/values/strings.xml
+++ b/app/src/main/res/values/strings.xml
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name" translatable="false">FairEmail</string>
+    <string name="app_name" translatable="false">Cloud Mail (Fairmail)</string>
     <string name="app_copyright" translatable="false">Copyright &#x24B8; 2018–2026 by M. Bokhorst</string>
     <string name="app_search">FairEmail search</string>
     <string name="app_send_self">Send to self</string>
